generators/app/templates/infrastructure/src/middleware/index.js

```diff
@@ -1,7 +1,9 @@
-<%_ if(withMultiTenancy){ _%>
+<%_ if(dataLayer == "knex") {_%>
```

Comment:
if dataLayer == "prisma" and withMultiTenancy == true, line no. 2 is going to be evaluated?